pkg/validationfile/fileformat.go

```diff
@@ -33,10 +33,12 @@ type ValidationFile struct {
 	ExpectedRelations blocks.ParsedExpectedRelations `yaml:"validation"`
 
 	// NamespaceConfigs are the namespace configuration protos, in text format.
+	//
```

Comment:
`gocritic` wants `Deprecated:` blocks to be separated from the rest of the doc comment.